src/components/Navbar.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+export default function Navbar() {
+  return (
+    <div className="navbar">
+      <h1 className="navTitle">Magician Calculator!</h1>
+      <nav className="navLinks">
+        <Link className="navItem" to="/about">Home</Link>
+        {' '}
+        |
+        {' '}
+        <Link className="navItem" to="/">Calculator</Link>
+        {' '}
+        |
+        {' '}
+        <Link className="navItem" to="/quote">quotes</Link>
+      </nav>
+    </div>
```

Comment:
- [ ] Upon first loading of the page, the calculator component shows in the middle of the page, what should display is the welcome page as in the screenshot below

Given design
![image](https://user-images.githubusercontent.com/81932478/200129329-95e703b7-8aea-463e-8e88-b5b1fa461b19.png)

Your design
![image](https://user-images.githubusercontent.com/81932478/200129349-6fb159c7-e359-4368-98f5-1becbdaafc6a.png)

- [ ] Also add the demrcating lines to the `navbar` as is in the design
![image](https://user-images.githubusercontent.com/81932478/200129757-93e935c4-9d94-4139-9130-a72777beec9c.png)